exporter/importables.go

```diff
@@ -27,11 +27,101 @@ import (
 )
 
 var (
-	//adlsGen2Regex = regexp.MustCompile(`^((?:abfs|wasb)s?)://([^@]+)@([^.]+)\.(?:[^/]+)(/.*)?$`)
 	adlsGen2Regex = regexp.MustCompile(`^(abfss?)://([^@]+)@([^.]+)\.(?:[^/]+)(/.*)?$`)
 	adlsGen1Regex = regexp.MustCompile(`^(adls?)://([^.]+)\.(?:[^/]+)(/.*)?$`)
+	wasbsRegex    = regexp.MustCompile(`^(wasbs?)://([^@]+)@([^.]+)\.(?:[^/]+)(/.*)?$`)
+	s3Regex       = regexp.MustCompile(`^(s3a?)://([^/]+)(/.*)?$`)
+	gsRegex       = regexp.MustCompile(`^gs://([^/]+)(/.*)?$`)
 )
 
+func generateMountBody(ic *importContext, body *hclwrite.Body, r *resource) error {
```

Comment:
Could you split this method into at least 3 smaller ones? it's almost 100 lines long